DOCS-319 - Alerts/Monitors tag update (#4303)
SumoLogic/sumologic-documentation: docs/alerts/monitors/settings.md

import useBaseUrl from '@docusaurus/useBaseUrl';

> The Monitors page allows you to view, create, manage, and organize your Monitors.

Author: ```suggestion
The Monitors page allows you to view, create, manage, and organize your Monitors. 
```

<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Monitors page, in the main Sumo Logic menu select **Manage Data > Monitoring > Monitors**.  

> <!--Kanso

Author: ```suggestion
<!--Kanso 
```

<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Monitors page, in the main Sumo Logic menu select **Manage Data > Monitoring > Monitors**.  
<!--Kanso 

> [**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**.

Author: ```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. 
```

  * **Export**. Provides JSON of the monitor, allowing you to transfer content within Sumo Logic by copying this JSON, then pasting it into the import dialog in the [Library](/docs/get-started/library) location you choose. This JSON format may change without notice. 
  * **Edit Permissions**.
  * **Delete**.
  * **Subscribe**.
  * **Copy Link**.<br/><img src={useBaseUrl('img/monitors/monitor-actions.png')} alt="monitor more actions" width="400"/>

## Tags

Adding a **Tag** to a monitor can help you categorize, search, filter, and correlate them with other data. These tags are key/value pairs that allow you to apply additional metadata to your monitors beyond their name and description. Example tag: `team=alerting`, where key is `team` and value is `alerting`.

### Syntax

- Tag keys cannot start with the prefixes `sumo.` or `_`

> - Tag keys can only contain letters, numbers, and/or the symbols `_`, `.`, `/`, `+`, `-`, `@`

Author: ```suggestion
- Tag keys must only contain letters, numbers, and/or the symbols `_`, `.`, `/`, `+`, `-`, `@`
```

   * Apply an existing tag by selecting the key and its possible values from the dropdown.
1. Click **Save**.

### What happens next?

Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.

> To view and edit tags, go to your alerts list page at **Manage Data > Monitoring > Monitors**, where you'll see a tags column. This page allows filtering by tags.

Reviewer: ```suggestion
To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->

Here you'll see a tags column. This page allows filtering by tags.
```

> <img src={useBaseUrl('img/alerts/monitors-alerts-list-tags.png')} alt="Alerts list page showing a column labeled 'Tags' next to alert details" />
> 
> You will also find your tags on the Alert Response page. Here's an example:
> 
> <img src={useBaseUrl('img/alerts/alert-response-tags.png')} alt="Alert Response page displaying graphs with tags shown in the tags section" />

Reviewer: ```suggestion
<img src={useBaseUrl('img/alerts/monitors-alerts-list-tags.png')} alt="Alerts list page showing a column labeled 'Tags' next to alert details" style={{border: '1px solid gray'}} width="800"  />

You will also find your tags on the Alert Response page. Here's an example:

<img src={useBaseUrl('img/alerts/alert-response-tags.png')} alt="Alert Response page displaying graphs with tags shown in the tags section" style={{border: '1px solid gray'}} width="800" />
```

   * Apply an existing tag by selecting the key and its possible values from the dropdown.
1. Click **Save**.

### What happens next?

> Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.

Author: ```suggestion
Tags will appear on all future alerts triggered by the same monitor. Tags will be applied to any past alerts triggered by this monitor.
```

> Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.

Reviewer 2: ```suggestion
Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor. If tags on a monitor are edited, that will not be reflected on previous alerts that were triggered by this monitor.
```

> To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->

Reviewer 2: navigating via `Manage Data > Monitoring > Monitors` will take the user to the Monitors Page and not the Alert List page as we have mentioned in the previous line.

> To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->

Reviewer 2: Also users cannot edit the tags on the Alerts. They can only see it and use them for filtering.

Although users can edit tags on the Monitor, the edited tags will not be updated on the alerts that were previously triggered. We should mention this too.

> To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->

Reviewer 2: ```suggestion
To view the tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> You can click on the tab with bell icon to visit the alert list page. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**. Kanso-->
```

Author: great catch, @jinkane
